hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -530,48 +531,40 @@ public SnapshotDiffCleanupService getSnapshotDiffCleanupService() {
   }
 
   /**
-   * Helper method to locate the end key with the given prefix and iterator.
+   * Helper method to perform operation on keys with a given prefix and iterator
    * @param keyIter TableIterator
    * @param keyPrefix DB key prefix String
    * @return endKey String, or null if no keys with such prefix is found
    */
-  private static String findEndKeyGivenPrefix(
+  private static void performOperationGivenPrefix(
       TableIterator<String, ? extends Table.KeyValue<String, ?>> keyIter,
-      String keyPrefix) throws IOException {
-
-    String endKey;
+      String keyPrefix, ThrowableFunction<Table.KeyValue<String, ?>,
```

Comment:
Please add `operationFunction` in java doc comment of `performOperationGivenPrefix`.

Reply:
done